src/sqs/queues.ts

```diff
@@ -9,12 +9,20 @@ const sqsQueues = {
 	backfill: new SqsQueue<BackfillMessagePayload>({ queueName: "backfill",
 		queueUrl: envVars.SQS_BACKFILL_QUEUE_URL,
 		queueRegion: envVars.SQS_BACKFILL_QUEUE_REGION,
-		longPollingIntervalSec: LONG_POLLING_INTERVAL_SEC}, backfillQueueMessageHandler),
+		longPollingIntervalSec: LONG_POLLING_INTERVAL_SEC,
+		timeoutSec: 10*60,
+		maxAttempts: 3
+	},
+	backfillQueueMessageHandler,
+	{handle: () => ({retryable: true})}
```

Comment:
For now adding trivial error handler. Will add an actual error handling for Push queue in the next PR

Reply:
In that case, wouldn't it be outside of the scope of this PR and should remove all error handling from this PR?

Reply:
Because the SQSQueue constructor requires one. 

I could potentially default it in the SQS constructor, however would like to make it explicit that we have an error handler (Hence in all our queues we'll use it)